### DISCUSSION TITLE
I think someone had a stroke when updating the Wiki...

### DISCUSSION THREAD

**Discussion by crypto-contributor** [Chia-Network/chia-blockchain#6994] [General]
c.f. https://github.com/Chia-Network/chia-blockchain/wiki/Connecting-the-UI-to-a-remote-daemon#set-the-chia_rootcccccclvjccfhidhuchgdgjghrtiiublugducdtlvucf

I think "cccccclvjccfhidhuchgdgjghrtiiublugducdtlvucf" is a typo :rofl: 

In all seriousness, this change was made during last update (https://github.com/Chia-Network/chia-blockchain/wiki/Connecting-the-UI-to-a-remote-daemon/_compare/b6d1360bad6262ab4b8c74336d465cb6e6b7181b...46db515b6909861dcc19806549eb685c2293ebf8#diff-ae61021d145cc886f398d31d9ac3b2aab4d0b5d0a8f4b47b70af087ee5e72058L36)

**Comment by lvcivs** [comment]
Cat walked over the keyboard, probably ;)

**Comment by loppefaaret** [comment]
hah! good find - it is our new AI roybotbot - it almost flawless, but we struggle a bit with the firmware still :)
